### PR TITLE
Moving standard library file

### PR DESCRIPTION
**Pull request by thisisjofrank**

### CODE REVIEW

File: runtime/fundamentals/standard_library.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+```ts
+import { copy } from "@std/fs";
+import { join } from "@std/path";
+
+await copy("foo.txt", join("dist", "foo.txt"));
+```
+
+Alternatively, you can import modules directly with the `jsr:` specifier:
+
+```json
```
Comment (thisisjofrank) [inline runtime/fundamentals/standard_library.md:52]:
```suggestion
```js
```

File: runtime/fundamentals/standard_library.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+import { copy } from "@std/fs";
+import { join } from "@std/path";
+
+await copy("foo.txt", join("dist", "foo.txt"));
+```
+
+Alternatively, you can import modules directly with the `jsr:` specifier:
+
+```js
+// This works without import map
```
Comment (thisisjofrank) [inline runtime/fundamentals/standard_library.md:53]:
```suggestion
```

File: runtime/_data.ts [hunk 3]
```diff
@@ -18,9 +18,11 @@ export const sidebar = [
     items: [
       "/runtime/fundamentals/ts_support/",
       "/runtime/fundamentals/permissions/",
+      "/runtime/fundamentals/standard_library/"
       "/runtime/fundamentals/debugging/",
       "/runtime/fundamentals/workspaces/",
       "/runtime/fundamentals/stability/",
+
```
Comment (marvinhagemeister) [inline runtime/_data.ts:25]: nit space, maybe a `deno fmt` was missing.

**Review by marvinhagemeister — APPROVED** [review]

File: runtime/_data.ts (hunk 3 above)
Comment (thisisjofrank) [inline runtime/_data.ts:25]:
```suggestion
```

**Pull request merged**